lib/Steps/Integrations/ReactNative.ts

```diff
@@ -265,14 +266,6 @@ export class ReactNative extends MobileProject {
     );
   }
 
-  private addZLibToXcode(proj: any): void {
-    proj.addPbxGroup([], 'Frameworks', 'Application');
-    proj.addFramework('libz.tbd', {
-      link: true,
-      target: proj.getFirstTarget().uuid,
-    });
-  }
-
```

Comment:
React Native is based on cocoapods since a long time and we get the dep for zlib there when installing the sdk